packages/dialtone-tokens/generate-themes.js

```diff
@@ -67,3 +67,25 @@ export default {
   fs.writeFileSync(filePath, content);
   console.log(`Generated ${fileName.replace('.js', '')} theme`);
 }
+
+/**
+ * Generate the universal high-contrast theme file for a specific mode
+ * @param {string} mode - The mode (light or dark)
+ */
+async function generateHighContrastTheme (mode) {
+  const fileName = `high-contrast-${mode}.js`;
+  const filePath = path.join(THEMES_OUTPUT_DIR, fileName);
+  const cssFileName = mode === 'light' ? 'contrast-high-light' : 'contrast-high-dark';
+
+  const content = `import Contrast from '@dialpad/dialtone-tokens/tokens-${cssFileName}.css?inline';
```

Comment:
concerned about this statement

```
import Contrast from '@dialpad/dialtone-tokens/tokens-${cssFileName}.css?inline'
```

Does the final file actually contain this statement? `?inline` is specific to vite/rollup and won't work on other bundlers.